### PR TITLE
Improve test Infrastructure [API-1510]

### PR DESCRIPTION
**Pull request by fatihozer0**
PR closed by Hazelcast automation as no activity (>3 months). Please reopen with comments, if necessary. Thank you for using Hazelcast and your valuable contributions

### CODE REVIEW

**Comment by yuce** [comment]
@fatihozer0 Could you fix the linter issues?

File: hazelcast/config.py [hunk 1]
```diff
@@ -435,6 +438,21 @@ def cluster_name(self, value: str) -> None:
 
         self._cluster_name = value
 
+    @property
```
Comment (mdumandag) [inline hazelcast/config.py:441]: Just a note, we should not add a configuration for this task. You should use the port you selected as the port of the addresses in the `cluster_members` config option

Comment (fatihozer0) [reply hazelcast/config.py:441]: I did not get what do you mean

Comment (mdumandag) [reply hazelcast/config.py:441]: We use the port from the cluster member addresses and should put that into there. 
```python
client = HazelcastClient(cluster_members=["localhost:12345"])
```
when you do your changes, you need to put the port you selected above, not into a new config you created here.

Comment (fatihozer0) [reply hazelcast/config.py:441]: Okay, reverting commit then.

**Pull request closed without merging**